odl/set/domain.py

```diff
@@ -232,6 +236,39 @@ def contains_set(self, other, tol=0.0):
             raise AttributeError('cannot test {!r} without `min()` and `max()`'
                                  'methods.'.format(other))
 
+    def contains_all(self, other):
+        """Test if all points defined by `other` are contained.
+
+        Parameters
+        ----------
+        other : object
+            Can be a single point, a `(d, N)` array where `d` is the number of
```

Comment:
I'll get them fixed. Doc is somewhat pre-done, I haven't built it. It's going to be the last point on the agenda.